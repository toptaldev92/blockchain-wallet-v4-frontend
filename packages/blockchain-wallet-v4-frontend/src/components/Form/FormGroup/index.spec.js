import React from 'react'
import FormGroup from './index.js'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

describe('FormGroup', () => {
  it('renders correctly', () => {
    const props = { margin: '20px' }
    const component = renderer.create(<FormGroup {...props} />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
