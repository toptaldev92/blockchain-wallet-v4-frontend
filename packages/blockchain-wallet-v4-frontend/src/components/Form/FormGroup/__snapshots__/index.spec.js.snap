// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormGroup renders correctly 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  margin-bottom: 20px;
}

.c0 > div {
  margin-right: 0px;
}

.c0 > div:last-child {
  margin-right: 0px;
}

.c0:last-child {
  margin-bottom: 0px;
}

<div
  className="c0"
/>
`;
